bellows/zigbee/zcl/__init__.py

```diff
@@ -77,13 +77,19 @@ def __init__(cls, name, bases, nmspc):
             for command_id, details in cls.server_commands.items():
                 command_name, schema, is_reply = details
                 cls._server_command_idx[command_name] = command_id
+        if hasattr(cls, 'client_commands'):
```

Comment:
Can you put the client_commands stuff in a separate PR, please?

Reply:
Actually, never mind ... this is OK as is.

Reply:
Okay